cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TemporalTest.scala

```diff
@@ -243,17 +226,17 @@ class TemporalTest extends DocumentingTest {
           p(
             """The following formats are supported for specifying time zones:
               |
-              |[options="header", width="85%"]
+              |[options="header", cols="<20,<30,<30,^10,^10"]
```

Comment:
```suggestion
              |[options="header", cols="<20,<30,<30,^15,^15"]
```
Or something like that, basically make the last two columns on the right slightly bigger. It renders as 
Supporte
d for
`DateTim`
`e`